src/OMSimulatorLua/OMSimulatorLua.c

```diff
@@ -1891,5 +1931,14 @@ DLLEXPORT int luaopen_OMSimulatorLua(lua_State *L)
   lua_pushnumber(L, oms_system_sc);
   lua_setglobal(L, "oms_system_sc");
 
+  lua_pushnumber(L, oms_signal_type_real);
+  lua_setglobal(L, "Real");
+  lua_pushnumber(L, oms_signal_type_integer);
+  lua_setglobal(L, "Integer");
+  lua_pushnumber(L, oms_signal_type_boolean);
+  lua_setglobal(L, "Boolean");
+  lua_pushnumber(L, oms_signal_type_string);
+  lua_setglobal(L, "String");
```

Comment:
I think we should keep the C names, e.g. `oms_signal_type_real` instead of `Real`.